workflows/sarek.nf

```diff
@@ -253,18 +254,60 @@ if (!params.dbsnp && !params.known_indels) {
     }
 }
-if (params.joint_germline && (!params.tools || !(params.tools.split(',').contains('haplotypecaller') || params.tools.split(',').contains('sentieon_haplotyper')))) {
-    error("The GATK's Haplotypecaller or Sentieon's Haplotyper should be specified as one of the tools when doing joint germline variant calling.) ")
+if (params.joint_germline && (!params.tools || !(params.tools.split(',').contains('haplotypecaller') || params.tools.split(',').contains('sentieon_haplotyper') || params.tools.split(',').contains('sentieon_dnascope')))) {
+    error("The GATK's Haplotypecaller, Sentieon's Dnascope or Sentieon's Haplotyper should be specified as one of the tools when doing joint germline variant calling.) ")
 }
 
-if (params.joint_germline && (!params.dbsnp || !params.known_indels || !params.known_snps || params.no_intervals)) {
-    log.warn "If GATK's Haplotypecaller or Sentieon's Haplotyper is specified, without `--dbsnp`, `--known_snps`, `--known_indels` or the associated resource labels (ie `known_snps_vqsr`), no variant recalibration will be done. For recalibration you must provide all of these resources.\nFor more information see VariantRecalibration: https://gatk.broadinstitute.org/hc/en-us/articles/5358906115227-VariantRecalibrator \nJoint germline variant calling also requires intervals in order to genotype the samples. As a result, if `--no_intervals` is set to `true` the joint germline variant calling will not be performed."
+if (
+    params.tools &&
+    (
+        params.tools.split(',').contains('haplotypecaller') ||
+        params.tools.split(',').contains('sentieon_haplotyper') ||
+        params.tools.split(',').contains('sentieon_dnascope')
+    ) &&
+    params.joint_germline &&
+    (
+        !params.dbsnp ||
+        !params.known_indels ||
+        !params.known_snps ||
+        params.no_intervals
+    )
+    ) {
+    log.warn("""If GATK's Haplotypecaller, Sentieon's Dnascope and/or Sentieon's Haplotyper is specified, \
+but without `--dbsnp`, `--known_snps`, `--known_indels` or the associated resource labels (ie `known_snps_vqsr`), \
+no variant recalibration will be done. For recalibration you must provide all of these resources.\nFor more information \
+see VariantRecalibration: https://gatk.broadinstitute.org/hc/en-us/articles/5358906115227-VariantRecalibrator \n\
+Joint germline variant calling also requires intervals in order to genotype the samples. \
+As a result, if `--no_intervals` is set to `true` the joint germline variant calling will not be performed.""")
 }
 
+if (params.tools &&
+    params.tools.split(',').contains('sentieon_dnascope') &&
+    params.joint_germline &&
+    (
+        !params.sentieon_dnascope_emit_mode ||
+        !params.sentieon_dnascope_emit_mode.split(',').contains('gvcf')
+    )
+    ) {
+    error("When using Sentieon Dnascope for joint-germline variant-calling the option `--sentieon_dnascope_emit_mode` has to include `gvcf`.")
+}
+
+if (params.tools &&
```

Comment:
@maxulysse @FriederikeHanssen : Can that kind of error-handling or "validation" also be handle by nf-validation?